.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
-.idea/
\ No newline at end of file
+.idea/
+dist/
+package-lock.json
```

Comment:
@IgorChaikin Its better to delete file from git history, before adding it to `package-lock.json`